server/management/appointments/appointments.php

```diff
@@ -49,7 +49,7 @@ function getAppointments($year) {
 				JOIN AppointmentTime ON Appointment.appointmentTimeId = AppointmentTime.appointmentTimeId
 				JOIN Site ON AppointmentTime.siteId = Site.siteId
 				JOIN Client ON Appointment.clientId = Client.clientId
-				JOIN SelfServiceAppointmentRescheduleToken ON Appointment.appointmentId = SelfServiceAppointmentRescheduleToken.appointmentId
+				LEFT JOIN SelfServiceAppointmentRescheduleToken ON Appointment.appointmentId = SelfServiceAppointmentRescheduleToken.appointmentId
```

Comment:
What's the reason for this change? Thanks~

Reply:
I needed the `token` in order to construct the unique URLs. The left join is because not every appointment has a token (in particular, the appointments that were created before the token infrastructure was built or ones that were created manually by us rather than through the site), but we still want all the appointments.